cmd/minikube/cmd/config/addons_list.go

```diff
@@ -93,62 +94,94 @@ var stringFromStatus = func(addonStatus bool) string {
 var printAddonsList = func(cc *config.ClusterConfig, printDocs bool) {
 	addonNames := slices.Sorted(maps.Keys(assets.Addons))
 	table := tablewriter.NewWriter(os.Stdout)
+	table.Options(tablewriter.WithHeaderAutoFormat(tw.On))
 
-	table.Options(
-		tablewriter.WithHeaderAutoFormat(tw.On),
-	)
-
-	// Create table header
-	var tHeader []string
-	if cc == nil {
-		tHeader = []string{"Addon Name", "Maintainer"}
-	} else {
-		tHeader = []string{"Addon Name", "Profile", "Status", "Maintainer"}
+	// Table header
+	header := []string{"Addon Name", "Maintainer"}
+	if cc != nil {
+		header = []string{"Addon Name", "Enabled", "Maintainer"}
 	}
 	if printDocs {
-		tHeader = append(tHeader, "Docs")
+		header = append(header, "Docs")
 	}
-	table.Header(tHeader)
+	table.Header(header)
+
+	var rows [][]string
 
-	// Create table data
-	var tData [][]string
-	var temp []string
 	for _, addonName := range addonNames {
 		addonBundle := assets.Addons[addonName]
+
 		maintainer := addonBundle.Maintainer
 		if maintainer == "" {
 			maintainer = "3rd party (unknown)"
 		}
+
```

Comment:
Why to you keep adding unrelated whitespace changes? I already commented about it in previous reviews several times.